tests/test_fetch.py

```diff
@@ -287,3 +288,8 @@ def test_query_caching(self):
 
         # reset cache directory state (will fail if purge was unsuccessful)
         os.rmdir(os.path.expanduser('~/dj_query_cache'))
+
+    def test_fetch_group_by(self):
+        # https://github.com/datajoint/datajoint-python/issues/914
+
+        Parent().fetch('KEY', order_by='name')
```

Comment:
Would you add an `assert` statement here to be specific on what we are expecting from this query used in this example?

Reply:
addressed this in commit ce16d0e